btas/generic/coupled_cp_als.h

```diff
@@ -142,15 +143,16 @@ namespace btas{
     /// \returns 2-norm
     /// error between exact and approximate tensor, -1 if calculate_epsilon =
     /// false && ConvClass != FitCheck.
-    double compute_PALS(std::vector<ConvClass> & converge_list, double RankStep = 0.5, int panels = 4,
-            int max_als = 20,bool fast_pI = false, bool calculate_epsilon = false, bool direct = true) override{
+    double compute_PALS(std::vector<ConvClass> &converge_list, double RankStep = 0.5, size_t panels = 4,
```

Comment:
I don't believe that ConvClass can be constant.  In the check I need to store and modify the previous fit values since the convergence is relative change in fit, not the actual fit.

Reply:
One way that I could allow ConvClass to be constant is to have the check function take a reference double that can store the previous iterations fit and rewrite the value with the current iterations fit. Would that be better?